pkg/fuzz/component/path.go

```diff
@@ -95,8 +96,14 @@ func (q *Path) Rebuild() (*retryablehttp.Request, error) {
 	lengthSplitted := len(q.value.parsed.Map)
 	for i := lengthSplitted; i > 0; i-- {
 		key := strconv.Itoa(i)
-		original := originalValues[key].(string)
-		new := q.value.parsed.Map[key].(string)
+
+		original := originalValues.GetOrDefault(key, "").(string)
```

Comment:
Shall we guard here using `fmt.Sprint(..)`, anyway if it's a string we will get the same value of `.(string)`?

Reply:
> Shall we guard here using `fmt.Sprint(..)` [...]

If the values are always strings (_or not?_), sticking with the type assertion might be preferable for performance & type safety reasons.

> [...] anyway if it's a string we will get the same value of `.(string)`?

Since the type of map is `any`, we still need to do a type assertion.

Reply:
But we should ensure that the assertion went correctly with:

```
a, ok := c.(string)
if !ok {
    continue 
}
```

what do you think?